code/datums/mob_stats.dm

```diff
@@ -238,6 +238,15 @@
 	name = STAT_VIG
 	desc = "Here, paranoia is nothing but a useful trait. Improves your ability to control recoil on guns, helps you resist insanity."
 
+/datum/stat/vivification
+	name = STAT_VIV
+	desc = "The body below the flesh can take only so much stimulation, takes a lot to train the body to handle drugs all be it helpful or harmful."
+
+/datum/stat/anatomy
+	name = STAT_ANA
+	desc = "The body itself, the more you know about how far you can push it, the more you can withstand being even at deaths door."
```

Comment:
what is "the body itself" doing here?
also it should be set off with a semicolon instead of a comma